store/extensions/GChatSaver/extension.json

```
@@ -31,7 +31,7 @@
         "clients": ["Flash"]
     },
 
-    "submissionDate": "12-02-2023 20:08:00",
-    "updateDate": "12-02-2023 20:08:00",
+    "submissionDate": "15-02-2023 13:25:00",

```

The submissiondate should remain unchanged
```suggestion
    "submissionDate": "12-02-2023 20:08:00",
```